pyfiglet/__init__.py

```diff
@@ -759,6 +767,21 @@ def getFonts(self):
         return self.Font.getFonts()
 
 
+def __get_ascii_color(color, isBackground):
```

Comment:
The standard is called [ANSI](https://en.wikipedia.org/wiki/ANSI_escape_code) -- [ASCII](https://en.wikipedia.org/wiki/ASCII) is something different; so `__get_ascii_color` should be called `get_ansi_color`, or perhaps better: `color_to_ansi`. You also make reference to ASCII in the error message and some variable names.

Reply:
😓 UPS!